models/intermediate/int_zendesk__ticket_schedules.sql

```diff
@@ -15,15 +15,29 @@ with ticket as (
   select *
   from {{ ref('stg_zendesk__schedule') }}
 
-
-), default_schedule_events as (
+), default_schedules as (
 -- Goal: understand the working schedules applied to tickets, so that we can then determine the applicable business hours/schedule.
 -- Your default schedule is used for all tickets, unless you set up a trigger to apply a specific schedule to specific tickets.
 
 -- This portion of the query creates ticket_schedules for these "default" schedules, as the ticket_schedule table only includes
 -- trigger schedules
+  select 
+    schedule_id,
+    source_relation
+  from (
+    
+    select
+      schedule_id,
+      source_relation,
+      row_number() over (partition by source_relation order by created_at) = 1 as is_default_schedule
+    from schedule
+
+  ) as order_schedules
+  where is_default_schedule
+
+), default_schedule_events as (
 
-{% if execute %}
+{# {% if execute %}
```

Comment:
Can you explain this update? It looks like the below variables are no longer used anymore. Is this true? And what was the need to remove the previous implementation?

Reply:
ah yes meant to delete this after confirming it worked -- i realized we didn't need the `run_query` actually, it was essentially just doing a very simple 1:N join. now it's doing a <number of unioned soures>:N join